docker/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+VERBOSE=FALSE
+VERBOSE_DOCKER_RUN=""
+VERBOSE_DOCKER_BUILD="-q"
+REMOVE_IMAGE=FALSE
+
+docker >/dev/null 2>&1 || { echo >&2 "Docker is required but it's not installed. Aborting."; exit 1; }
```

Comment:
Minor nit-pick but I don't like that we're blindly running whatever `docker` is in $PATH here just to check that it's available. Can we just `which docker > /dev/null || {...}` here instead?